packages/server/src/AccountsServer.js

```diff
@@ -302,6 +321,57 @@ export class AccountsServer {
     const res = await this.db.setProfile(userId, { ...user.profile, ...profile });
     return res;
   }
+
+  async sendVerificationEmail(userId: string, address: string): Promise<void> {
+    const user = await this.db.findUserById(userId);
```

Comment:
You should probably extract the code from this line to L338 to another function because it is repeating on the `sendResetPasswordEmail` method